www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+* **Data Type**: Vector data.
+
+![World Settlement Footprint (WSF) Evolution example](images/wsfevo.png "World Settlement Footprint (WSF) Evolution example")
+
+**[World Settlement Footprint (WSF) Evolution - Landsat-5/7 Dataset (WSFEVO)](https://download.geoservice.dlr.de/WSF_EVO/)**
+
+* **Coverage:** 2,323,907 polygons
+* **License**: CC-BY 4.0
+* **Description**: Raw raster data, being converted into vector data. Place counts represent cell count from raster > vector conversion.
+* **Data Type**: Vectorized raster data
+
+**[Socioeconomic Data and Applications Center (SEDAC) / Center for International Earth Science Information Network (CIESIN)](https://sedac.ciesin.columbia.edu/data/set/india-india-village-level-geospatial-socio-econ-1991-2001/data-download)**
+
+* **Coverage:** 627,199 polygons
+* **License**: CC-BY
+* **Description**: Used for locality polygons across India (1991 & 2001 vintage). Population data may be gathered from this source.
+* **Data Type**: Vector data.
+
+**[Sentinel-2 10m Land Use/Land Cover (ESRI, Microsoft, National Geographic)](https://www.arcgis.com/apps/instant/media/index.html?appid=fc92d38533d440078f17678ebc20e8e2)**
+
+* **Coverage:** 3,377,461 polygons
+* **License**: CC-BY 4.0
+* **Description**: Raw raster data, being converted into vector data. Place counts represent cell count from raster > vector conversion.
+* **Data Type**: Vectorized raster data
+
+**[Global Human Settlement Layer (GHSL)](https://ghsl.jrc.ec.europa.eu/ghs_stat_ucdb2015mt_r2019a.php)**
+
+* **Coverage:** 3,242,056 polygons
+* **License**: CC-BY 4.0
+* **Description**: Raw raster data, being converted into vector data. Place counts represent cell count from raster > vector conversion. Being used to “optimize” urban localities; if GHSL points fell within one of the raster data sources, a place would be considered valid.
+* **Data Type**: Vectorized raster data
+
+
+**Honorable mention:** After the Karmashapes data initiative was completed and as we were importing Karamashapes into Who’s On First, the [SHRUG](https://www.devdatalab.org/shrug) version 2 (Socioeconomic High-resolution Rural-Urban Geographic Platform for India) dataset from Development Data Lab was released. SHRUG includes 500,000 villages and 8,000 towns with data attributes covering a span of 25 years, all linked together with a set of common geographic identifiers. While the SHURG v2 CC-BY-NC-SA license limits it for non-commercial projects or direct import into Who’s On First, this could be an area of future collaboration.
+
+![Combined Karmashapes input datasets](images/baralu1.png "Combined Karmashapes input datasets")
+_(above) All data sources for Baralu/Barahalu/Barahallu overlaid in QGIS_
+
+![Example of final place data for Baralu](images/baralu2.png "Example of final place data for Baralu")
+_(above) The final polygon and label position for the place of Baralu_
```

Comment:
To: `The final polygon and label position for the Baralu locality record (with alternate names of Barahalu and Barahallu)`

Reply:
Hmm - good call, I should have walked through this example to the Explorer.

We did import name translations, but I'd have to look into the full Karmashapes dataset to figure out why names weren't added in this case. I can come back to this later today.

Reply:
FWIW: I don't think this should hold up publishing the blog post.